Tests/MapboxMapsTests/Camera/BasicCameraAnimatorTests.swift

```diff
@@ -85,14 +85,14 @@ final class BasicCameraAnimatorTests: XCTestCase {
         XCTAssertNotNil(animator?.transition)
         XCTAssertEqual(animator?.transition?.toCameraOptions.zoom, 10)
         XCTAssertEqual(delegate.cameraAnimatorDidStartRunningStub.invocations.count, 1)
-        XCTAssertTrue(delegate.cameraAnimatorDidStartRunningStub.parameters.first === animator)
+        XCTAssertTrue(delegate.cameraAnimatorDidStartRunningStub.invocations.map(\.parameters).first === animator)
```

Comment:
```suggestion
        XCTAssertTrue(delegate.cameraAnimatorDidStartRunningStub.invocations.first.parameters === animator)
```

Please apply this change in all cases that are like this.